src/test/java/com/easypost/ShipmentTest.java

```diff
@@ -794,4 +795,22 @@ public void testBuyShipmentWithEndShipperId() throws EasyPostException {
 
         assertNotNull(boughtShipment.getPostageLabel());
     }
+
+    /**
+     * Tests that we retrieve time-in-transit data for each of the Rates of a Shipment.
+     *
+     * @throws EasyPostException when the request fails.
+     */
+    @Test
+    public void testRetrieveEstimatedDeliveryDate() throws EasyPostException {
+        vcr.setUpTest("retrieve_estimated_delivery_date");
+
+        Shipment shipment = vcr.client.shipment.create(Fixtures.basicShipment());
+
+        List<EstimatedDeliveryDate> estimatedDeliveryDates = vcr.client.shipment
+                .retrieveEstimatedDeliveryDate(shipment.getId(), "2023-04-28");
```

Comment:
suggestion: let's move the date to a fixture (I will do this on my already merged ones when we do version bumps)